omnibus/config/software/datadog-agent.rb

```diff
@@ -80,7 +80,7 @@
     command "inv -e rtloader.clean"
     command "inv -e rtloader.make --python-runtimes #{py_runtimes_arg} --install-prefix \"#{install_dir}/embedded\" --cmake-options '-DCMAKE_CXX_FLAGS:=\"-D_GLIBCXX_USE_CXX11_ABI=0 -I#{install_dir}/embedded/include\" -DCMAKE_C_FLAGS:=\"-I#{install_dir}/embedded/include\" -DCMAKE_INSTALL_LIBDIR=lib -DCMAKE_FIND_FRAMEWORK:STRING=NEVER'", :env => env
     command "inv -e rtloader.install"
-    bundle_arg = bundled_agents.map { |k| "--bundle #{k}" }.join(" ")
+    bundle_arg = bundled_agents ? bundled_agents.map { |k| "--bundle #{k}" }.join(" ") : "--bundle agent"
```

Comment:
I'm not sure I understand the expected logic, if nothing is added to `bundled_agent` then the default behavior is  to bundle only the agent.

So far so good, but this seem to imply that the agent needs to be explicitly enabled to be built, and currently on linux `bundled_agents` will contain `process-agent` `security-agent` and `system-probe`, which should cause `--bundle agent` not to be passed to the omnibus task and the agent not to be built.

Could you clarify what the expected behavior is?

Reply:
To disable the bundling completely, one has to use `inv agent.build --bundle agent`.
If you want to bundle other agents, you need to either:
- use the defaults specified in the "invoke" files
- explicitely define the list (this is what we do here).